.github/workflows/_comps-workflow.yml

```diff
@@ -39,25 +49,37 @@ jobs:
         uses: actions/checkout@v4
 
       - name: Clone required Repo
+        id: get-yaml-path
         run: |
           cd ${{ github.workspace }}/.github/workflows/docker/compose
           # service=$(echo ${{ inputs.service }} | cut -d'_' -f1)
+          if [[ "${{ inputs.mode }}" == "CD" ]]; then
+            if [ -f "${{ github.workspace }}/.github/workflows/docker/compose/${{ inputs.service }}-compose-CD.yaml" ]; then
+              if [ -f "${{ github.workspace }}/.github/workflows/docker/compose/${{ inputs.service }}-compose.yaml" ]; then
+                sed -n '/services:/,/^$/p' ${{ github.workspace }}/.github/workflows/docker/compose/${{ inputs.service }}-compose-CD.yaml |sed '1d' >> ${{ github.workspace }}/.github/workflows/docker/compose/${{ inputs.service }}-compose.yaml
```

Comment:
No need to combine CI/CD list.

Reply:
If don't combine CI/CD list, copying during the image build phase will result in errors.